readingdata/Person.json

```
@@ -88,6 +88,12 @@
         "required": false,
         "pattern": "(^$|NULL|S[0-9]{2}|unknown)"
       }
+    }, {
+      "name": "narrative_age",
+      "constraints": {
+        "required": false,
+        "pattern": "(^$|NULL|[0-3]|unknown)"
```

```suggestion
        "pattern": "(^$|NULL|[0-3]|unknown)"
```
```suggestion
        "pattern": "(^$|NULL|0[0-3])"
```
I would suggest making `00` = `unkown` so to not mix text with numbers as possible field values, also use two digits (eg. `01`) so we have more room to maneuver down the road